src/components/dropdown-filter/dropdown-filter.stories.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import {
+  text,
+  select,
+  boolean,
+  number
+} from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+import { State, Store } from '@sambego/storybook-state';
+import ImmutableHelper from '../../utils/helpers/immutable';
+import OptionsHelper from '../../utils/helpers/options-helper';
+import notes from './notes.md';
+import DropdownFilter from './dropdown-filter';
+
+const store = new Store({
+  value: ''
+});
+
+const create = (evt, component) => component.state.filter;
+
+// Shared Props
+const onChange = (evt) => {
+  store.set({ value: evt.target.value });
+  action('change')();
```

Comment:
can you pass event as the second argument?

Reply:
👍